package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "test": "hardhat --network localhost test",
-    "lint": "find contracts/ -iname *.cairo -exec .venv/bin/cairo-format -c {} +",
+    "lint": "find rewaave/ -iname *.cairo -exec .venv/bin/cairo-format -c {} +",
     "compile": "hardhat starknet-compile --cairo-path 'node_modules/@joriksch/oz-cairo:contracts/src/starknet'",
```

Comment:
I'm not sure renaming `contracts` to `rewaave` is such a good idea. using that name as repository name is was better IMO. if you drop the folder `contracts`, readers who don't the know our terminology won't know where the contracts are located when looking at the root. it always feels better when you understand in a glance how a repository is structured.

Reply:
agree^